apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.html.tsx

```diff
@@ -82,224 +105,235 @@ export function InstructorInstanceAdminSettings({
+                  type="submit"
+                  class="btn btn-primary mb-2"
+                  name="__action"
+                  value="update_configuration"
+                  disabled={!isDirty}
                 >
-                  Edit course instance configuration
+                  Save
+                </button>
+                <button
+                  id="cancel-button"
+                  type="button"
+                  class="btn btn-secondary mb-2 ms-2"
+                  onClick={() => reset()}
+                >
+                  Cancel
+                </button>
+                <p class="mb-0">
+                  <a
+                    data-testid="edit-course-instance-configuration-link"
+                    href={encodePathNoNormalize(
+                      `${urlPrefix}/${navPage}/file_edit/${infoCourseInstancePath}`,
+                    )}
+                  >
+                    Edit course instance configuration
+                  </a>{' '}
+                  in <code>infoCourseInstance.json</code>
+                </p>
+              </>
+            ) : (
+              <p class="mb-0">
+                <a href={`${urlPrefix}/${navPage}/file_view/${infoCourseInstancePath}`}>
+                  View course instance configuration
                 </a>{' '}
                 in <code>infoCourseInstance.json</code>
               </p>
-            </>
-          ) : (
-            <p class="mb-0">
-              <a href={`${urlPrefix}/${navPage}/file_view/${infoCourseInstancePath}`}>
-                View course instance configuration
-              </a>{' '}
-              in <code>infoCourseInstance.json</code>
-            </p>
-          )}
-        </form>
-      </div>
-      <div class="card-footer d-flex flex-wrap gap-2">
-        <form name="copy-course-instance-form" method="POST">
-          <input type="hidden" name="__csrf_token" value={csrfToken} />
+            )}
+          </form>
+        </div>
+        <div class="card-footer d-flex flex-wrap gap-2">
           <button
-            type="submit"
-            name="__action"
-            value="copy_course_instance"
+            type="button"
             class="btn btn-sm btn-primary"
+            onClick={() => setShowCopyModal(true)}
           >
             <i class="fa fa-clone" /> Make a copy of this course instance
           </button>
-        </form>
-        <button
-          type="button"
-          class="btn btn-sm btn-primary"
-          data-bs-toggle="modal"
-          data-bs-target="#deleteCourseInstanceModal"
-        >
-          <i class="fa fa-times" aria-hidden="true" /> Delete this course instance
-        </button>
+          <button
+            type="button"
+            class="btn btn-sm btn-primary"
+            data-bs-toggle="modal"
+            data-bs-target="#deleteCourseInstanceModal"
+          >
+            <i class="fa fa-times" aria-hidden="true" /> Delete this course instance
```

Comment:
Can we hide the copy and delete buttons if the user doesn't have course edit permissions?